pkg/controller/operconfig/mtu_probe.go

```diff
@@ -134,13 +144,16 @@ func (r *ReconcileOperConfig) deleteMTUProber(ctx context.Context, infra *bootst
 	return nil
 }
 
-func renderMTUProber(infra *bootstrap.InfraStatus) ([]*uns.Unstructured, error) {
+func renderMTUProber(infra *bootstrap.InfraStatus, proxyStatus configv1.ProxyStatus) ([]*uns.Unstructured, error) {
```

Comment:
nit: make proxyStatus part of the bootstrap.InfraStatus, don't add it as another function argument.

Reply:
Fixed